packages/docs/src/locales/en/en.json

```diff
@@ -661,16 +661,18 @@
         "canBeFocused": "If false, then input can not be focused using mouse or keyboard."
```

Comment:
Keyboard navigation didn't really work for me.
![image](https://user-images.githubusercontent.com/5394573/147934635-0c0e428f-bb03-4a5f-af98-c29c6d9c08be.png)

Reply:
now if there is no manulaInput mode, onDropdown open timePicker is focused